common/time_utils.py

```
@@ -137,17 +137,28 @@ def format_iso_8601_to_ymd_hms(time_str: str) -> str:
+    if not time_str:
+        return ""
     try:
-        dt = parser.isoparse(time_str)
+        # ``dateutil.parser.isoparse`` accepts every ISO 8601 form the LLM
+        # extraction prompt can plausibly emit, including ordinal dates
+        # (``2024-001T12:00:00``) that stdlib ``datetime.fromisoformat``
+        # rejects. Strip the tz offset so ``strftime`` formats the wall
+        # clock unchanged, matching the legacy return path.
+        dt = parser.isoparse(time_str).replace(tzinfo=None)
         return dt.strftime("%Y-%m-%d %H:%M:%S")
     except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)